### PR TITLE
Amls 5186 - HVD: application can be submitted with missing payment date

### PR DESCRIPTION
**Pull request by Dandyboyo2**
HVD: application can be submitted with missing payment date

## How Has This Been Tested?
Manual
Unit

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/models/hvd/Hvd.scala [hunk 1]
```diff
@@ -82,12 +82,18 @@ case class Hvd (cashPayment: Option[CashPayment] = None,
...
     Logger.debug(s"[Hvd][isComplete] $this")
     this match {
-        case Hvd(Some(_), Some(pr), _, Some(_), Some(_), Some(true), Some(_), Some(_), _, _, true)
-          if pr.items.forall(item => item != Alcohol && item != Tobacco) => true
-        case Hvd(Some(_), Some(_), Some(_), Some(_), Some(_), Some(true), Some(_), Some(_), _, _, true) => true
-        case Hvd(Some(_), Some(pr), _, Some(_), Some(_), Some(false), _, Some(_), _, _, true)
-          if pr.items.forall(item => item != Alcohol && item != Tobacco) => true
-        case Hvd(Some(_), Some(_), Some(_), Some(_), Some(_), Some(false), _, Some(_), _, _, true) => true
+        case Hvd(Some(cp), Some(pr), _, Some(_), Some(_), Some(true), Some(_), Some(_), _, _, true)
+          if pr.items.forall(item => item != Alcohol && item != Tobacco) & cp.isCashPaymentsComplete => true
```
Comment (owen-opencast) [inline app/models/hvd/Hvd.scala]: Did you intend to use a single & here? Would usually use double
https://stackoverflow.com/questions/8691693/bitwise-operator-and-single-ampersand

**Comment by owen-opencast** [comment]
Just a small query RE: ampersands.... it will work either way,  but usually double ampersands would be used.

File: app/models/hvd/Hvd.scala (hunk 1 above)
Comment (Dandyboyo2) [reply app/models/hvd/Hvd.scala]: Thanks Owen, I have amended and pushed that now.

Comment (owen-opencast) [reply app/models/hvd/Hvd.scala]: Think you missed one :)

**Pull request merged**